sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeBeforeApiSuccessTest.java

```diff
@@ -144,6 +149,13 @@ private void showUpdateSetupFailedDialog(ArgumentCaptor<DialogInterface.OnClickL
         PreferencesStorage.remove(PREFERENCE_KEY_UPDATE_SETUP_FAILED_MESSAGE_KEY);
     }
 
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        when(mContext.getSharedPreferences(PREFERENCES_NAME_MOBILE_CENTER, Context.MODE_PRIVATE)).thenReturn(mMobileCenterPreferencesStorage);
```

Comment:
Looks like you defined mMobileCenterPreferencesStorage in Abstract test already, you can remove this method, the field in this class and probably move the the when to parent set up.

Either that or review why mMobileCenterPreferencesStorage is also in Abstract test.